airflow/api_connexion/openapi/v1.yaml

```diff
@@ -441,6 +441,41 @@ paths:
         '401':
           $ref: '#/components/responses/Unauthenticated'
 
+    patch:
+      summary: Update DAGs
+      x-openapi-router-controller: airflow.api_connexion.endpoints.dag_endpoint
+      operationId: patch_dags
+      tags: [DAG]
+      parameters:
+        - $ref: '#/components/parameters/PageLimit'
+        - $ref: '#/components/parameters/PageOffset'
+        - $ref: '#/components/parameters/FilterTags'
+        - $ref: '#/components/parameters/UpdateMask'
+        - name: only_active
+          in: query
+          schema:
+            type: boolean
+            default: true
+          required: false
+          description: |
+            Only update active DAGs.
+        - name: dag_id_pattern
+          in: query
+          schema:
+            type: string
+          required: false
+          description: |
+            If set, only update DAGs with dag_ids matching this pattern.
```

Comment:
We should have these schemas in the components/parameters section so it can be reused if need arises

Reply:
RequestBody is needed too

Reply:
@ashb I had modified both the `get` and `patch` bulk endpoints. `~` can be supplied to return all dags if needed. 
I had done it this way since `dag_id_parameter` is a reusable parameter as per this thread. I am thinking to define `dag_id_pattern` separately in both bulk endpoints and define it as required in `patch` and optional in `get`. WDYT?